[doc] 5.4.3.3  Deploy Metadata Servers

### Type of report
   - [x] bug
   - [ ] feature request

### Source Document
5.4.3.3  Deploy Metadata Servers:

https://susedoc.github.io/doc-ses/master/single-html/ses-deployment/#deploy-cephadm-day2-service-mds

### Summary
PLACEMENT_STRING parameter is not explained

### Expected Results
Optional parameter could be removed or should be described

Error message for simple string

------------
ceph fs volume create CEPHFS_NAME my_ceph_placement
Error EINVAL: Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 34, in assert_valid_host
    assert p.match(part), 'name component must include only a-z, 0-9, and -'
AssertionError: name component must include only a-z, 0-9, and -

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/ceph/mgr/mgr_module.py", line 1188, in _handle_command
    return self.handle_command(inbuf, cmd)
  File "/usr/share/ceph/mgr/volumes/module.py", line 367, in handle_command
    return handler(inbuf, cmd)
  File "/usr/share/ceph/mgr/volumes/module.py", line 372, in _cmd_fs_volume_create
    return self.vc.create_fs_volume(vol_id, placement)
  File "/usr/share/ceph/mgr/volumes/fs/volume.py", line 87, in create_fs_volume
    return create_volume(self.mgr, volname, placement)
  File "/usr/share/ceph/mgr/volumes/fs/operations/volume.py", line 70, in create_volume
    return create_mds(mgr, volname, placement)
  File "/usr/share/ceph/mgr/volumes/fs/fs_util.py", line 41, in create_mds
    placement=PlacementSpec.from_string(placement))
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 366, in from_string
    host_pattern=host_patterns[0] if host_patterns else None)
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 169, in __init__
    self.validate()
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 276, in validate
    h.validate()
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 143, in validate
    assert_valid_host(self.hostname)
  File "/usr/lib/python3.6/site-packages/ceph/deployment/service_spec.py", line 36, in assert_valid_host
    raise ServiceSpecValidationError(e)
ceph.deployment.service_spec.ServiceSpecValidationError: name component must include only a-z, 0-9, and -

--- a/xml/deploy_cephadm.xml
+++ b/xml/deploy_cephadm.xml
@@ -1512,6 +1512,12 @@ placement:
      automatically deployed when you create the &cephfs;. To create a &cephfs;,
      first create MDS servers by applying the following specification:
     </para>
+     <note>
+      <para>
+        Ensure you have at least two pools created before applying the following
+        specification.
+      </para>
+    </note>
 <screen>
 service_type: mds
 service_id: <replaceable>CEPHFS_NAME</replaceable>
@@ -1524,16 +1530,7 @@ placement:
     <para>
      After MDSs are functional, create the &cephfs;:
     </para>
-<screen>&prompt.cephuser;ceph fs volume create <replaceable>CEPHFS_NAME</replaceable> <replaceable>PLACEMENT_STRING</replaceable></screen>
-     <note>
-       <para>
-         The <replaceable>PLACEMENT_STRING</replaceable> is an optional string
-         identifying the hosts. For example, the following placement string
-         deploys &ganesha; daemons on nodes <literal>host1</literal> and
-         <literal>host2</literal> (one daemon per host): 
-       </para>
-<screen>&prompt.cephuser;ceph fs volume create volume_1 “host1,host2”</screen>
-     </note>
+<screen>ceph fs new <replaceable>CEPHFS_NAME</replaceable> <replaceable>metadata_pool</replaceable> <replaceable>data_pool</replaceable></screen>
    </sect3>
    <sect3 xml:id="deploy-cephadm-day2-service-ogw">
     <title>Deploy &ogw;s</title>

--- a/xml/deploy_cephadm.xml
+++ b/xml/deploy_cephadm.xml
@@ -1514,8 +1514,8 @@ placement:
     </para>
      <note>
       <para>
-        Ensure you have at least two pools created before applying the following
-        specification.
+        Ensure you have at least two pools, one for &cephfs; data and one for &cephfs;
+        metadata, created before applying the following specification.
       </para>
     </note>
 <screen>

--- a/xml/deploy_cephadm.xml
+++ b/xml/deploy_cephadm.xml
@@ -1525,6 +1525,15 @@ placement:
      After MDSs are functional, create the &cephfs;:
     </para>
 <screen>&prompt.cephuser;ceph fs volume create <replaceable>CEPHFS_NAME</replaceable> <replaceable>PLACEMENT_STRING</replaceable></screen>
+     <note>
+       <para>
+         The <replaceable>PLACEMENT_STRING</replaceable> is an optional string
+         identifying the hosts. For example, the following placement string
+         deploys &ganesha; daemons on nodes <literal>host1</literal> and
+         <literal>host2</literal> (one daemon per host): 
+       </para>
+<screen>&prompt.cephuser;ceph fs volume create volume_1 “host1,host2”</screen>
+     </note>
    </sect3>
    <sect3 xml:id="deploy-cephadm-day2-service-ogw">
     <title>Deploy &ogw;s</title>

